recipes/torchvision-extra-decoders/recipe.yaml

```
@@ -0,0 +1,79 @@
+context:
+  name: torchvision-extra-decoders
+  version: "0.0.2"
+  # https://github.com/pytorch-labs/torchvision-extra-decoders/issues/22
+  gitsha: b6d9e5c8fae88f9980a9faaf0b8c71787ca5b193
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/pytorch-labs/torchvision-extra-decoders/archive/${{ gitsha }}.zip
+  sha256: c67624b96688ba3f0550a71093f40a9f1fc9dfd8d182ffd8de703d805042758d
+  patches:
+    - 0001-Always-return-library-found-for-conda-forge.patch
+    - 0002-Allow-to-specify-git-sha-through-environment-variabl.patch
+
+build:
+  skip:
+    # Upstream only claims linux support. so having linux + osx is better already
+    # Dont' have time to debug their build system today
+    # Evetnaully one gets the error
+    # LINK : error LNK2001: unresolved external symbol PyInit_extra_decoders_lib
+    - win
+  script:
+    content:
+      - python -m pip install . -vv
+    env:
+      BUILD_GIT_SHA: ${{ gitsha }}
+      BUILD_VERSION: ${{ version }}
+  number: 0
+
+requirements:
+  build:
+    - ${{ stdlib('c') }}
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+        - pytorch
+  host:
+    - python
+    - pip
+    - setuptools
+    - libavif
+    # Add a "naked" libheif so that the global pinning is picked up
+    - libheif
+    # Add a constraint on the build time configuration of libheif to avoid gpl code
+    - libheif [build=lgpl*]
+    - libtorch
+    - pytorch
+  run:
+    - python
+
+tests:
+  - python:
+      imports:
+        - torchvision_extra_decoders
+      pip_check: true
+  - requirements:
+      run:
+        - pip
+    script:
+      - pip list
+      - python -c 'import torchvision_extra_decoders; print(torchvision_extra_decoders.__version__)'
+      - if: unix
+        then: pip list | grep torchvision-extra-decoders | grep ${{ version }}
+
+about:
+  homepage: https://github.com/pytorch-labs/torchvision-extra-decoders
+  summary: 'An extension of TorchVision for decoding AVIF and HEIC images.'
```

```suggestion
  summary: 'An extension of TorchVision for decoding AVIF and HEIC images'
```

Sorry for nitpicking, but it's not a proper sentence.

---

^_^